docs/source/open-software-week/index.md

```diff
@@ -3,16 +3,19 @@
 We are excited to announce our inaugural **NIU Open Software Week**, taking
 place in **August 2025** in **London, UK**. This event will bring together researchers, developers, and users of open-source software for some hands-on training, community-building and hacking.
 
-:::{admonition} Save the Date!
+:::{admonition} Applications are open. Apply for your spot now by filling in [this Google form](https://forms.gle/2UAAzikhSgYArZpX7)!
 :class: info
 
 **August 11-15 2025**, [Sainsbury Wellcome Centre](https://maps.app.goo.gl/CzWFFjXJZwX87aMj6)
 
-- **April 16th 2025**: Registration opens
+- **April 16th 2025**: Applications open
 - **May 16th 2025**: Travel grant application deadline
-- **June 30th 2025**: Registration closes
+- **June 30th 2025**: Applications close
+- **July 14th 2025**: Applicants are notified of acceptance
```

Comment:
What is the reasoning for closing the travel grant applications if we don't let applicants know until July anyway? I.e. what happens between May 16th and July 30th?

I would also remove the specific date for when we will get back to people, and just put e.g. "July 2025", there may be delays, or we might be able to get back to applicants quicker.

Reply:
IIRC we were thinking something like "we will use that period to target advertising to fill up remaining places with local community members that don't need travel grants"... but maybe we should simplify.

Reply:
I would just have one deadline, maybe between the two (so that those needing funding find out early enough). We can then reopen for only non-funded places later on if there's space.